_posts/2023-03-17-introduction-to-discrete-event-simulations-in-healthcare/introduction-to-discrete-event-simulations-in-healthcare.Rmd

```diff
@@ -0,0 +1,112 @@
+---
+title: "Introduction to Discrete Event Simulations in Healthcare"
+description: |
+  An introduction to Discrete Events Simulations (DES) in healthcare with an overview of the use of DES for modelling outpatient services appointments at Providence Healthcare.
+  
+author:
+  - name: Kevin Wang, Neil Mistry
+    url: https://www.linkedin.com/in/mingkun-wang/, https://www.linkedin.com/in/neilmistry/
+date: 2022-09-23
+output:
+  distill::distill_article:
+    self_contained: false
+draft: true
+category: post-miscellaneous
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+# Discrete Events Simulation at Providence Outpatient Care & Services Covid-19 Recovery scheduling
+
+
+Discrete Events Simulations (DES) are a method of modeling the operation of real world systems digitally.  It does this by modeling a series of events in time.  Each event is ordered by time of occurance, and the system is advanced to each event in the sequence, meanwhile updating the state of the system.  This is done so that we can test hypothetical changes to the system and their impacts on metrics we are interested in, uncover bottlenecks or process challenges and make overall assessments of the system.
+
+This type of simulation is particularly useful at modeling a system with many uncertain or probabalistic steps (such as you don't know when your next customer or patient will arrive, and you don't know how long and what resources are required to help that customer or patient.)  In sytems where there a complex number of steps, many different resources, inputs and outputs, DES may be one of the only possible ways to model.
```

Comment:
```suggestion
This type of simulation is particularly useful at modeling a system with many uncertain or probabalistic steps (such as you don't know when your next customer or patient will arrive, and you don't know how long and what resources are required to help that customer or patient.)  In systems where there are a complex number of steps, many different resources, inputs and outputs, DES may be one of the only possible ways to model.
```

Reply:
Adding to this: I think it'd be better to unpack that parenthetical. So something like

This type of simulation is particularly useful at modeling a system with many uncertain or probabalistic steps. For examples as you don't know when the next patient will arrive, you don't know how long [until what?], and what resources are required to help that patient.